s.py

```diff
@@ -0,0 +1,229 @@
+import csv
+from dataclasses import dataclass
+from typing import Optional, List, Union, Dict
+
+import requests
+from bs4 import BeautifulSoup
+
+
+@dataclass
+class Everyone:
+    id: int
+    profile_url: str
+    username: Optional[str]
+    name: Optional[str]
+    github_username: Optional[str]
+    twitter_username: Optional[str]
+    location: Optional[str]
+    joined_at: Optional[Union[str, int]]
+    linked_in: Optional[str]
+    email: Optional[str]
+    work: Optional[str]
+    stackOverflow: Optional[str]
+    instagram: Optional[str]
+    facebook: Optional[str]
+    youtube: Optional[str]
+    education: Optional[str]
+    github_url: Optional[str]
+    twitter_url: Optional[str]
+    work_status: Optional[str]
+
+
+def get_profile_url() -> List[Everyone]:
+
+    code = requests.get(f"https://dev.to/api/users/1")
+
+    devs = code.json()
+
+    partial_dev_data: List[Everyone] = []
+
+    id: int = devs.get("id")
+    username: str = devs.get("username")
+    name: str = devs.get("name")
+    twitter_username: str = devs.get("twitter_username")
+    github_username: str = devs.get("github_username")
+    location: str = devs.get("location")
+    joined_at: Union[str, int] = devs.get("joined_at")
+    profile_url: str = f"https://dev.to/{username}"
+
+    dev_to_be_parsed: Everyone = Everyone(
+        id=id,
+        profile_url=profile_url,
+        username=username,
+        name=name,
+        github_username=github_username,
+        github_url="",
+        twitter_url="",
+        twitter_username=twitter_username,
+        location=location,
+        joined_at=joined_at,
+        linked_in="",
+        email="",
+        work="",
+        stackOverflow="",
+        instagram="",
+        facebook="",
+        youtube="",
+        education="",
+        work_status="",
+    )
+    if username:
+        partial_dev_data.append(dev_to_be_parsed)
+
+    #print(partial_dev_data)
```

Comment:
When asking for a review remove all the useless comments from your code. We don't want some random commented out code on master